src/lib/capacityEngine.js

```diff
@@ -89,15 +58,15 @@ function calculatePoolUtilizationRate(products) {
 function calculateProductCapacity(
   store,
   productId,
-  { poolId = null, periodSeconds, withPools = false, now, assets, assetRates },
+  { poolId = null, period, now, assets, assetRates, withPools = true },
```

Comment:
`periodSeconds` renamed to `period`

Reply:
also notice we kept `withPools` param internally, this is so we can skip `capacityPerPool` field as it does not make sense to have it on other endpoints. (`calculateProductCapacity` is shared amongst the other endpoints)

Reply:
then would it make sense to drop the `capacityPerPool` in the controller and avoid passing this param down here?

Reply:
`capacityPerPools` is constructed on the same iteration when we calculate the capacity (i.e. aggregatedData). 

hence the logic for calculating `capacityPerPools` is coupled in `calculateProductCapacity` as its much more efficient to do it that way. 

if we want to optimise it further I can pass down the `withPools` down and if its false skip the logic for calculating `capacityPerPools`.

see `calculateProductDataForTranche` for logic

Reply:
@shark0der will merge this for now. as I'm keen to get this to staging for the FE.

if you want further changes I'll just do it in a separate PR, if that is ok.